### PR TITLE
Parameterize home directory contents

### PR DESCRIPTION
**Pull request by jamesconant**
Give greater control over the contents of the user's home directory
via declared attributes.

### CODE REVIEW

File: attributes/user.rb [hunk 1]
```diff
@@ -4,4 +4,6 @@
   user['group'] = 'ajones'
   user['gid'] = '1000'
   user['home'] = '/home/ajones'
+  user['home']['manage?'] = true
```
Comment (http-418) [inline attributes/user.rb:7]: This needs to default to either `false` or `nil`, or it will break all existing users.

File: recipes/user.rb [hunk 2]
```diff
@@ -32,13 +32,16 @@
...
     shell '/bin/bash'
-    manage_home true
+    manage_home user['home']['manage?']
   end
 
-  directory user['home'] + '/bin' do
-    owner user['name']
-    group user['group']
-    mode 0700
+  user['home']['directories'].each do |dir|
```
Comment (http-418) [inline recipes/user.rb:38]: I'm not interested in merging this part.  Write a wrapper recipe and manage extra directories based on attributes.

File: attributes/user.rb (hunk 1 above)
Comment (http-418) [reply attributes/user.rb:7]: (additionally you may wish to consider using `chef_rewind` to add the manage attribute yourself without any upstream changes)

**Comment by jamesconant** [comment]
Canceling request to merge...

**Pull request closed without merging**